lua/nvim-tree/actions/copy-paste.lua

```diff
@@ -212,9 +212,15 @@ function M.print_clipboard()
 end
 
 local function copy_to_clipboard(content)
-  vim.fn.setreg("+", content)
-  vim.fn.setreg('"', content)
-  return a.nvim_out_write(string.format("Copied %s to system clipboard! \n", content))
+  if M.use_sys_clipboard == true then
+    vim.fn.setreg("+", content)
+    vim.fn.setreg('"', content)
```

Comment:
i think you need to `"\""` to fix stylua issues.
You can run stylua locally to, it should report these errors.